### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by samuliheljo**

### CODE REVIEW

File: pages/developers/getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: Getting started
+order: 4
+---
+
+First you should decide "what do you want to get started with?". You have multiple options:
+
+1. Test drive Digitransit-ui
+2. Use Digitransit APIs in your application
+3. Make modifications to Digitransit-ui web app
+4. Running other APIs on your local machine
```
Comment (tuukka) [inline pages/developers/getting-started/index.md]: Why "other" APIs instead of just APIs?

File: pages/developers/routing-api/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+> http://www.opentripplanner.org/
+
+Routing API enables developers to query routes and timetable related information using either REST or GraphQL interfaces.
+
+### Static and realtime information
+
+OpenTripPlanner APIs provide access to static and realtime information. Routing and timetable data is based on static GTFS and it is enriched by realtime information for those departures that have realtime information available. This means that results returned by OpenTripPlanner always contain realtime information should it be available.
+
+## Service architecture
+![Architecture](./architecture.jpg)
```
Comment (tuukka) [inline pages/developers/routing-api/index.md]: OSM missing from the OTP inputs in the diagram?

File: pages/developers/realtime-hsl-api/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Realtime HSL - API
+---
+
+## Dataflow
+
+![Dataflow](./dataflow.png)
+
+Heavy lifting of realtime data is done in other systems. Digitransit user interface integrates to MQTT datasources in order to read realtime data from vehicles. In Helsinki city region realtime service, vehicle data is sent once every 30 seconds. Our realtime server works as a cache for MQTT datasource thus enabling user interface to query "old" realtime information and show that data immediately to user.
```
Comment (tuukka) [inline pages/developers/realtime-hsl-api/index.md]: s/sent once every 30 seconds/sent once every 1-30 seconds where available/

File: pages/developers/routing-data/index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+title: Routing - Data
+---
+
+## Dataflow
+![Dataflow](./dataflow.png)
```
Comment (tuukka) [inline pages/developers/routing-data/index.md]: The diagram should be updated to remove Kalkati.

File: pages/developers/runtime_environment/index.md [hunk 5]
```diff
@@ -13,7 +14,7 @@ https://github.com/HSLdevcom/digitransit-deploy/blob/master/roles/run/templates/
 
 Gray circle in the image above marks what lives inside compose. HAProxy works as a load balancer for the compose.
 
-Why do we need HAProxy in front of the compose? We actually run multiple composes in production. One called "Active" on "Passive". 
+Why do we need HAProxy in front of the compose? We actually run multiple composes in production. One called "Active" on "Passive".
```
Comment (tuukka) [inline pages/developers/runtime_environment/index.md]: s/ on /, one /

File: pages/developers/routing-data/index.md (hunk 4 above)
Comment (samuliheljo) [inline pages/developers/routing-data/index.md]: I suggest that we do this in https://digitransit.atlassian.net/browse/DT-617

**Pull request merged**